DCF_Demo.md

```diff
@@ -309,14 +357,18 @@ az iot hub invoke-device-method -n [name-of-iothub] -d [name-of-device] --mn "ke
 </details>
 <br>
 
-### From Blob example
+## From Blob example
```

Comment:
>From Blob example [](http://example.com/codeflow?start=3&length=17)

do you plan on covering default packages in this demo?

Reply:
No, I'll cover that in the PR that change the capability name to positional, in this case the default will be clearly marked by the `*` in the name.